lib/taskana-core-test/src/test/java/acceptance/task/query/TaskQueryImplAccTest.java

```diff
@@ -149,6 +149,15 @@ void should_ReturnAllTasksAsTaskadmin_When_NoAccessItemForAWorkbasketExists() {
               taskSummary1, taskSummary2, taskSummary3, taskSummary4, taskSummary5);
     }
 
+    @WithAccessId(user = "admin")
+    @Test
+    void should_CountAllTasksFromWorkbasketAsAdmin_When_NoAccessItemForWorkbasketExists() {
+      long result =
+          taskService.createTaskQuery().workbasketIdIn(wbWithoutPermissions.getId()).count();
+
+      assertThat(result).isEqualTo(3);
+    }
+
```

Comment:
why did you add this test? TSK-1909 is only a performance optimization. Shouldn't there be already a test which verifies this?